x/group/client/testutil/tx.go

```diff
@@ -2306,6 +2200,197 @@ func (s *IntegrationTestSuite) TestTxLeaveGroup() {
 	}
 }
 
+func (s *IntegrationTestSuite) TestExecProposalsWhenMemberLeavesOrIsUpdated() {
+	val := s.network.Validators[0]
+	clientCtx := val.ClientCtx
+
+	weights := []string{"1", "1", "2"}
+	accounts := s.createAccounts(3)
+	testCases := []struct {
+		name                    string
+		indexOfMemberThatLeaves int
+		votes                   []string
+		members                 []string
+		malleate                func(groupID string) (*cobra.Command, []string)
+		expectErr               bool
+		errMsg                  string
+		respType                proto.Message
+	}{
+		{
+			"member leaves while all others vote yes",
+			0,
+			[]string{"VOTE_OPTION_YES", "VOTE_OPTION_YES", "VOTE_OPTION_YES"},
+			accounts,
+			func(groupID string) (*cobra.Command, []string) {
+				indexOfMemberThatLeaves := 0
```

Comment:
what do you think about this shorter var name?
```suggestion
				leavingMemberIdx := 0
```